src/openforms/forms/models/form_variable.py

```diff
@@ -97,6 +86,7 @@ def create_for_formstep(self, form_step: "FormStep") -> List["FormVariable"]:
                     is_sensitive_data=component.get("isSensitiveData", False),
                     source=FormVariableSources.component,
                     data_type=get_component_datatype(component),
+                    initial_value=component.get("defaultValue"),
```

Comment:
Make a `get_default_value` in utils for formio documenting that Formio has the empty value and default value concepts + links to github